lib/ansible/modules/database/postgresql/postgresql_pg_hba.py

```diff
@@ -684,7 +680,7 @@ def main():
         add_file_common_args=True,
         supports_check_mode=True
     )
-    if not HAS_IPADDRESS:
+    if IPADDRESS_IMP_ERR is not None:
         module.fail_json(msg=missing_required_lib('psycopg2'), exception=IPADDRESS_IMP_ERR)
```

Comment:
This is an unrelated change but I think this line should be
```suggestion
         module.fail_json(msg=missing_required_lib('ipaddress'), exception=IPADDRESS_IMP_ERR)
```

Reply:
I think it's fine to touch this line as well.